botnim/benchmark/assistant_loop.py

```diff
@@ -64,30 +92,87 @@ def assistant_loop(client: OpenAI, assistant_id, question=None, thread=None, not
                 step_details: ToolCallsStepDetails = step.step_details
                 for tool_call in step_details.tool_calls:
                     if tool_call.type == 'function':
-                        print('TOOL', tool_call.id, tool_call.function.name, tool_call.function.arguments)
-                        notes.append(f'{tool_call.function.name}({tool_call.function.arguments})')
```

Comment:
why did you remove this functionality?

Reply:
I did not consider it was needed for other parts. I've restored the original functionality for console output and notes collection while keeping the new file logging